composer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "dnadesign/silverstripe-elemental-skeletons",
 	"description": "Elemental Archetypes",
-	"type": "silverstripe-module",
+	"type": "silverstripe-vendormodule",
```

Comment:
With this change we'll also need to "expose" any CSS and Javascript files in the "extra" section of composer.json

Reply:
All good, no requirements